x-pack/plugins/apm/public/components/app/service_overview/service_overview_errors_table/index.tsx

```diff
@@ -147,14 +147,16 @@ export function ServiceOverviewErrorsTable({ serviceName }: Props) {
                 start,
                 end,
                 numBuckets: 20,
-                groupIds: JSON.stringify(
-                  items.map(({ groupId: groupId }) => groupId).sort()
-                ),
                 offset:
                   comparisonEnabled && isTimeComparison(offset)
                     ? offset
                     : undefined,
               },
+              body: {
+                groupIds: JSON.stringify(
+                  items.map(({ groupId: groupId }) => groupId).sort()
```

Comment:
Hey i know the `.sort` was there before, but i don't understand why it's necessary since the groupIds are being used with `keyBy` mapping them to an object anyways?

Reply:
I sort it to cache the request. If there are only 5 items in the list (no pagination is available) and the user clicks to sort the table by another column, the result of the detailed statistics will be cached.